app/MMB/src/scenes/tripOverview/Timeline.js

```diff
@@ -0,0 +1,63 @@
+// @flow strict
+
+import * as React from 'react';
+import { View, FlatList } from 'react-native';
+import { StyleSheet, Color } from '@kiwicom/mobile-shared';
+
+const lineWidth = 2;
+const circleSize = 12;
+
+type Props = {|
+  +data: React.Node[],
+|};
+
+export default class Timeline extends React.Component<Props> {
+  renderItem = ({ item, index }: {| +item: React.Node, +index: boolean |}) => {
+    const isLast = this.props.data.length === index + 1;
+    const lastItemStyle = {};
+
+    if (!isLast) {
+      lastItemStyle.borderColor = Color.ink.light;
+      lastItemStyle.borderStartWidth = lineWidth;
+    }
+
+    return (
+      <View style={styles.rowContainer}>
+        <View style={[styles.item, lastItemStyle]}>
+          <View style={{ marginBottom: 20 }}>{item}</View>
```

Comment:
Is it necessary to use inline style here?

Reply:
Moved... 🙄